### DISCUSSION TITLE
https://maxkim-j.github.io/posts/local-dev-server

### DISCUSSION THREAD

**Discussion by giscus** [MaxKim-J/maxkim-j.github.io#101] [Announcements]
# https://maxkim-j.github.io/posts/local-dev-server

dev 명령어를 치고 localhost로 접속하면 일어나는 일

https://maxkim-j.github.io/posts/local-dev-server

**Comment by hg-pyun** [comment]
유익한 포스트 감사합니다.

**Comment by sulmoJ** [comment]
오 재밌는데요?

**Reply by MaxKim-J** [reply]
굳
